api/bots_api/bots_test_lib.py

```diff
@@ -28,6 +28,32 @@
 class BotTestCase(TestCase):
     bot_name = ''  # type: str
 
+    def get_bot_message_handler(self):
+        # type: () -> Any
+        # message_handler is of type 'Any', since it can contain any bot's
+        # handler class. Eventually, we want bot's handler classes to
+        # inherit from a common prototype specifying the handle_message
+        # function.
+        bot_module = os.path.join(current_dir, "bots",
+                                  self.bot_name, self.bot_name + ".py")
+        lib_module = get_lib_module(bot_module)
+        return lib_module.handler_class()
+
+    def setUp(self):
```

Comment:
Those are standard methods of python's `unittest` class, they have to be named like this.